src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -74,7 +74,7 @@
                                         x:Uid="Globals_DefaultTerminal"
                                         x:Load="false">
                     <ComboBox x:Name="DefaultTerminal"
-                              ItemsSource="{x:Bind State.Settings.DefaultTerminals, Mode=OneWay}"
+                              ItemsSource="{x:Bind State.Settings.DefaultTerminals}"

```

Comment:
Hmm.. why the change here? I guess we never reload this in practice...

Reply:
When I started testing my changes, the SUI was crashing on load and with @carlos-zamora I was able to determine that I made a mistake with my `DefaultTerminals()` implementation. After fixing my issue only this change remained. I believe this change to be correct as `DefaultTerminals` is unable to post updates (it's a pull-based API and cannot "push" updates).
But I do want to minimize the PR complexity and would thus be entirely happy to revert any change we deem unnecessary.

Reply:
Yep it refreshes `_State`.